grails-app/services/au/org/ala/profile/hub/ProfileService.groovy

```diff
@@ -366,6 +370,10 @@ class ProfileService {
         result
     }
 
+    def getPublicationJson(String pubId){
```

Comment:
Not sure why this method is required if it's just calling the other getPublications()?